.github/workflows/k8s-ci.yml

```diff
@@ -328,9 +328,19 @@ jobs:
       - name: Init Hermit
         run: ./bin/hermit env -r >> $GITHUB_ENV
 
+      - name: Set Kind Configuration
+        if: matrix.test-target == 'aws'
+        run: |
+          echo "KIND_CONFIG=kind-mono" >> $GITHUB_ENV
+
+      - name: Set Kind Configuration
+        if: matrix.test-target != 'aws'
+        run: |
+          echo "KIND_CONFIG=kind-multi" >> $GITHUB_ENV
```

Comment:
Can we do it in one step?

Reply:
@olegsu github actions do not support "if else", I can implement inline script for this functionality.